go.mod

```diff
@@ -431,3 +434,8 @@ replace github.com/microcosm-cc/bluemonday => github.com/microcosm-cc/bluemonday
 
 // this is a fork which addresses a performance issues due to go routines.
 replace dubbo.apache.org/dubbo-go/v3 => dubbo.apache.org/dubbo-go/v3 v3.0.3-0.20230118042253-4f159a2b38f3
+
+// Uncomment for local development for testing with changes in the components-contrib && kit repositories.
+// Don't commit with this uncommented!
+//
+// replace github.com/dapr/kit => ../kit
```

Comment:
Why was this added to the release branch?

Reply:
i just copy/pasted my changes from the other PR. Will come back and fix these bc didn't realize this also bumped go versions until i looked over my own pr. Give me a sec pls :)